gitgraph

```diff
@@ -12,6 +13,12 @@ QUIET=0
 
 ###############################################################################
 #
+
+TMP_HIGH="/tmp/$$-tmp.high"
+TMP_DOT="/tmp/$$-tmp.dot"
+TMP_SVG="/tmp/$$-out.svg"
+TMP_PNG="/tmp/$$-out.png"
+
```

Comment:
All temporary files was removed in tip of master. Maybe useful to resolve #1. Let's think about it